stable/wordpress/templates/NOTES.txt

```diff
@@ -6,6 +6,14 @@
   export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
   echo http://$NODE_IP:$NODE_PORT/admin
 
+{{- if .Values.ingress.enabled }}
```

Comment:
IMO this should be the first condition, and the NodePort one should be in an `else`.